theano/tensor/nnet/tests/test_abstract_conv.py

```diff
@@ -266,15 +291,18 @@ def test_all(self):
                     self.tcase(i, f, ds, db, dflip, provide_shape)
                 except SkipTest as e:
                     skipped = e
-            for s in self.subsamples:
-                for b in self.border_modes:
-                    try:
-                        self.tcase(i, f, s, db, dflip, dprovide_shape)
-                    except SkipTest as e:
-                        skipped = e
+            for fd in self.filters_dilations:
+                for s in self.subsamples:
+                    for b in self.border_modes:
+                        try:
+                            self.tcase(i, f, s, db, dflip,
+                                       dprovide_shape, fd)
```

Comment:
Hello, 
I've tried to run the test in `theano/gpuarray/tests/test_abstractconv.py` and found that tests failing because of a small mistake here. For making the tests pass, I had to remove this parameter `fd`. I don't know if this is the correct solution and would like to know from you people. The tests `TestCorrConv2d`, which uses this test as its parent class has a `tcase` method with a keyword argument fd. Would it be apt to move the `tcase` method to this class ?
cc : @lamblin @nouiz